pipdb.js

```diff
@@ -0,0 +1,114 @@
+require('buffertools').extend();
+
+var dbIndices = {};
+var dbTypes = {};
+
+var dbEntryType = ['Bool', 'Int8', 'UInt8', 'Int32', 'UInt32', 'Float', 'String', 'List', 'Dictionary'];
+
```

Comment:
We actually have these setup in [pipboylib as a parser](https://github.com/rgbkrk/pipboylib/blob/master/lib/parser.js) thanks to @nelix. Can we bring more of this type of functionality there, supplementing where possible?